test/specs/SendKeysSpec.ts

```diff
@@ -0,0 +1,49 @@
+import { assert } from 'chai';
+import { BrowserUtils } from '../../src/commons/BrowserUtils';
+import { SpecialKeys } from '../../src/enums/SpecialKeys';
+import { describeCommon } from '../TestHelper';
+
+const TEXT_INPUT: string = "//*[@id='SendKeys_input__text']";
+const text: string = 'Cloudinary';
+
+/**
+ * wdio-allure-ts sendKeys tests
+ */
+describeCommon('Send Keys', () => {
+  it('String array ', () => {
+    $(TEXT_INPUT).clearValue();
+    $(TEXT_INPUT).click();
+    BrowserUtils.sendKeys(text.split(''));
+    assert.equal($(TEXT_INPUT).getValue(), text);
+  });
+
+  it('Add text', () => {
+    $(TEXT_INPUT).clearValue();
+    $(TEXT_INPUT).click();
+    BrowserUtils.sendKeys(text.split(''));
+    $(TEXT_INPUT).click();
+    BrowserUtils.sendKeys(text.split(''));
```

Comment:
why do you have double input on every test?

Reply:
To check add text instead of new text list setValue

Reply:
just remove second click

Reply:
fixed